protostar/testing/cheatcodes/expect_call_cheatcode.py

```diff
@@ -32,25 +25,25 @@ def build(self) -> Callable[[int, str, list[int]], Callable]:
         return self.expect_call
 
     def expect_call(
-        self, raw_address: RawAddress, fn_name: str, calldata: list[int]
+        self, raw_address: RawAddress, function_name: str, calldata: list[int]
```

Comment:
Those changes seem out of scope of this PR

Reply:
I am not sure if we even have to unify those intefeaces, It could have been adressed separately, but if you feel it is not worth it you can leave it

Reply:
It's better to do it, otherwise, we'd end up with two different structures/sets of methods for the same purpose. For the reference, you can check out [protostar/starknet/cheatable_cached_state.py](https://github.com/software-mansion/protostar/pull/1363/commits/dce29d380746999059899f807331f7f29dc849d5#diff-8d74f2401d78312e7801aa75441a7f0268fc107e46778325d4354d83908bd104): `self.expected_contract_calls` and `self.expected_contract_calls_cairo`